fusion_estimator:
  Initial_State_X:  0.0
  Initial_State_Y:  0.0
  Initial_State_Z:  0.1
  Initial_State_Roll:  0
  Initial_State_Pitch: 0
  Initial_State_Yaw:   -0.13
  urdf_file_name: "resources/unitree_aliengo/urdf/aliengo_generated_edited.urdf"
  urdf_file_package: "ocs2_robotic_assets"
  urdf_BaseName: "base"
  urdf_HipJointName: "_HAA"
  urdf_HipLengthName: "_HFE"
  urdf_ThighLengthName: "_KFE"
  urdf_CalfLengthName: "_fOOT_fixed_2"
  urdf_FootLengthName: "_FOOT_Dummy"
  # urdf_file_name: "/resources/unitree_aliengo/urdf/aliengo_generated_edited.urdf"
  # urdf_file_package: "ocs2_robotic_assets"
  Sensor_Signal_Source_1: "/robot_description/anymal/low_state"
  Sensor_Signal_Source_2: "/odom"
  frequency: 400